src/app/containers/Ad/Amp/index.jsx

```diff
@@ -89,40 +102,48 @@ export const AMP_ACCESS_FETCH = service => {
 };
 
 // eslint-disable-next-line react/prop-types
-const AmpAd = ({ service }) => {
+const AmpAd = () => {
+  const { ads, dir, script, service } = useContext(ServiceContext);
+  const advertisementLabel = pathOr(false, ['advertisementLabel'], ads);
+
   return (
-    <>
-      <Helmet>
-        {AMP_ADS_JS}
-        {AMP_ACCESS_JS}
-        {AMP_ACCESS_FETCH(service)}
-      </Helmet>
-      <div
-        amp-access="toggles.ads.enabled AND geoIp.advertiseCombined"
-        amp-access-hide="true"
-      >
-        <StyledAd>
-          <amp-ad {...ampAdPropsMobile({ service })}>
-            <amp-img
-              placeholder
-              width="320"
-              height="50"
-              src="https://via.placeholder.com/320x50"
-              layout="responsive"
-            />
-          </amp-ad>
-          <amp-ad {...ampAdPropsDesktop({ service })}>
-            <amp-img
-              placeholder
-              width="970"
-              height="250"
-              src="https://via.placeholder.com/970x250"
-              layout="responsive"
-            />
-          </amp-ad>
-        </StyledAd>
-      </div>
-    </>
+    <FullWidthWrapper>
+      <StyledWrapper>
+        <Helmet>
+          {AMP_ADS_JS}
+          {AMP_ACCESS_JS}
+          {AMP_ACCESS_FETCH(service)}
+        </Helmet>
+        <div
+          // amp-access="toggles.ads.enabled AND geoIp.advertiseCombined"
+          amp-access-hide="true"
+        >
+          <StyledAd>
+            <StyledParagraph script={script} service={service} dir={dir}>
```

Comment:
We noticed during the home team meeting that in the current AMP implementation, the advertisement text is actually a link:

```
<a class="bbccom_text" href="https://www.bbc.com/privacy/cookies/international/" tabindex="-1">Advertisement</a>
```

@JonBeeb, @jimjohnsonrollings, shall we replicate this?

Reply:
Following conversations with @JonBeeb, we'll make the label to be a link that points to `https://www.bbc.com/privacy/cookies/international/` for all services